src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/external-references/GeneExternalReferences.scss

```diff
@@ -14,7 +14,7 @@
 
 .geneDetails {
   display: grid;
-  grid-template-columns: [gene-symbol] min-content 10px [stable-id] auto;
+  grid-template-columns: [gene-symbol] minmax(min-content, max-content) 10px [stable-id] auto;
```

Comment:
Sorry, I keep getting confused by `minmax`. Do you know how `minmax(min-content, max-content)` will behave here, and is it any different from `auto`?

Reply:
I tried `auto` but I was getting something like this:
![Screenshot 2020-11-26 at 15 18 50](https://user-images.githubusercontent.com/3085815/100367946-de218400-2ffa-11eb-858d-111b05b07584.png)


`minmax(min-content, max-content)`:

![Screenshot 2020-11-26 at 15 19 01](https://user-images.githubusercontent.com/3085815/100367967-e7aaec00-2ffa-11eb-91f7-bffa8da55403.png)

Reply:
Ah! I see. Just tried, and it would have worked with `[gene-symbol] auto [stable-id] 1fr;`

I don't have particularly strong feelings about the minmax — see for yourself whether you prefer that or the variant with auto, but `10px` though — that should be grid-column-gap, shouldn't it?

Reply:
By the way, do you remember why we are using a grid for this element? Why not just an inline-block next to an  inline? Or not a flex with flex-wrap?

Reply:
>Why not just an inline-block next to an inline? 

Not sure! The overview section uses two spans. I guess I might have tried to replicate what was on the XD where both the values are always on the same level.